src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/FilesTab.java

```diff
@@ -318,7 +318,9 @@ public void setTargetBundle(EntityBundle bundle) {
 		breadcrumb.configure(bundle.getPath(), EntityArea.FILES);
 		view.clearActionMenuContainer();
 		//Preview
-		view.setPreviewVisible(isFile);
+		if (!bundle.getFileHandles().isEmpty()) {
```

Comment:
If Folder, it used to call setPreviewVisible(false).  With this code change it wouldn't (so if you go to a file page, and then to a folder, it would likely still show the preview panel.